tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -206,3 +206,64 @@ def test_positive_resource_vm_power_management(
             session.computeresource.vm_poweron(name, rhev_data['vm_name'])
         assert session.computeresource.vm_status(
             name, rhev_data['vm_name']) is not status
+
+
+@tier2
+def test_positive_ec2_edit_resource_description(session):
+    """Edit EC2 Compute Resource with another description
+
+    :id: f75544b1-3943-4cc6-98d1-f2d0fbe7244d
+
+    :expectedresults: resource updated successfully and has new description
+
+    :CaseLevel: Integration
+    """
+    name = gen_string('alpha')
+    description = gen_string('alpha')
+    new_description = gen_string('alpha')
+    assert description != new_description
```

Comment:
I would not fail the test just because our randomization mechanism did a poor job :)

Reply:
yeah. Otherwise, we need to add similar validation across all other ~3k tests

Reply:
Yeah, you are right. We need to fix it as some time. I'll keep it there and created https://github.com/SatelliteQE/robottelo/issues/6488 for your proposal.

Reply:
can't you just append some character to the end of the new string so you're sure it is different?